### PR TITLE
pull out commtrack view calls

### PR DESCRIPTION
**Pull request by dannyroberts**
(more of the same, wait for tests)

### CODE REVIEW

File: corehq/apps/commtrack/models.py [hunk 1]
```diff
@@ -2,10 +2,18 @@
...
+
 from couchdbkit.exceptions import ResourceNotFound
-from dimagi.ext.couchdbkit import *
 from django.db import transaction
 from django.utils.translation import ugettext as _
+from django.dispatch import receiver
+from django.db import models
+from django.db.models.signals import post_save, post_delete
+
+from corehq.apps.commtrack.dbaccessors import \
```
Comment (dimagimon) [inline corehq/apps/commtrack/models.py]: F401 'get_supply_point_case_by_location_id' imported but unused

**Comment by dannyroberts** [comment]
probably shouldn't merge this without tests

File: corehq/apps/commtrack/models.py [hunk 2]
```diff
@@ -2,10 +2,17 @@
...
 from couchdbkit.exceptions import ResourceNotFound
-from dimagi.ext.couchdbkit import *
 from django.db import transaction
 from django.utils.translation import ugettext as _
+from django.dispatch import receiver
+from django.db import models
+from django.db.models.signals import post_save, post_delete
+
+from corehq.apps.commtrack.dbaccessors import get_supply_point_case_by_location
+from dimagi.ext.couchdbkit import *
```
Comment (esoergel) [inline corehq/apps/commtrack/models.py]: :+1:

**Comment by esoergel** [comment]
Keep this stuff coming!

**Pull request merged**